framework/Models/PostProcessors/DataClassifier.py

```diff
@@ -40,7 +40,7 @@ class cls.
       @ Out, inputSpecification, InputData.ParameterInput, class to use for
         specifying input of cls.
     """
-    inputSpecification = super(DataClassifier, cls).getInputSpecification()
```

Comment:
please open an issue for all the PP to add description of the parameters. We need to expand the automatic documentation coverage

Reply:
see #1525

Reply:
Please update the issue including that the _readMoreXML must be removed once the process is complete